### ISSUE TITLE
incorrect reading of numbers in English

### ISSUE THREAD

**Issue by williamh** [espeak-ng/espeak-ng#999]
Hi,

the default English voice for espeak-ng 0.50 reads numbers incorrectly. For example, it reads "245" as "two hundred and forty five". The "and" shouldn't be there; it should just read "two hundred forty five".

Thanks,

William

**Comment by williamh** [comment]
After a quick look around on the web, I have to change this to a question.
Is the default voice supposed to be US English? If so, it should drop the "and" when reading numbers. If not, feel free to close this, and sorry for the noise.

Thanks,

William

**Comment by hozosch** [comment]
no, the default one is British. There is the US voice though, which actually does drop the and. So, this can be closed I think.

**Comment by williamh** [comment]
Is there a way as a packager I can change the default voice at build time based on the end user's locale?

**Comment by hozosch** [comment]
That I don't know. I need to pass to @jaacoppi, @rhdunn or @valdisvi.

**Comment by jaacoppi** [comment]
The language files for English are stored in: espeak-ng-data/lang/gmw/

Technically you could either
1) rename "en" to "en-GB" and rename "en-US" to "en". That would 
2) change the settings in "en"
Both ways might break ISO 639 naming convention so I'm not sure if they should be used.

Another solution is to leave espeak-ng unmodified and modify the system. Make an alias so that "espeak-ng" actually calls "espeak-ng -v en-US"

**Comment by jaacoppi** [comment]
https://github.com/espeak-ng/espeak-ng/commit/d80f1a80a2f6b651b8d0e2ca3036209e81194b5d makes it easier to set a default voice. Just change here: https://github.com/espeak-ng/espeak-ng/blob/d80f1a80a2f6b651b8d0e2ca3036209e81194b5d/src/include/espeak-ng/espeak_ng.h#L39

It seems to work for most cases. However, there are still checks like `if (tr->translator_name == L('e', 'n')`. Some of them are related to default language, some are English specific.  I don't know if the tests cover them already.  Since translator_name is an int and ESPEAKNG_DEFAULT_VOICE is a string I need to figure out how to compare then in a simple way.
